audiocall/src/main/java/com/voximplant/demos/kotlin/audio_call/stories/call/OngoingCallViewModel.kt

```diff
@@ -61,6 +72,18 @@ class OngoingCallViewModel : ViewModel(), IAudioDeviceEventsListener {
             }
         }
 
+        audioCallManager.callState.observeForever { callState ->
```

Comment:
we should avoid `observeForever` as a call has its own life time

Reply:
logic moved to addSource